frontend/src/views/WheresMyGeneV2/components/CellInfoSideBar/index.tsx

```diff
@@ -206,32 +233,57 @@ function CellInfoSideBar({
                   sdsStyle="minimal"
                   sdsType="secondary"
                   isAllCaps={false}
+                  data-testid={MARKER_SCORE_TOOLTIP_TEST_ID}
                 >
                   <StyledIconImage src={questionMarkIcon} />
                 </TooltipButton>
               </Tooltip>
             </DivTableCell>
-          </DivTableHead>
-          <DivTableLegend>
-            <DivTableCell>
-              <CopyGenesButton
-                onClick={handleCopyGenes}
-                sdsType="primary"
-                sdsStyle="minimal"
-                isAllCaps={false}
-                startIcon={<Icon sdsIcon="copy" sdsSize="s" sdsType="button" />}
+            <DivTableCell align data-testid="marker-genes-table-specificity">
+              {TABLE_HEADER_SPECIFICITY}
+              <Tooltip
+                sdsStyle="dark"
+                placement="bottom"
+                width="default"
+                className="fmg-tooltip-icon"
+                arrow
+                onOpen={() => setHoverStartTime(Date.now())}
+                onClose={handleMarkerScoreHoverEnd}
```

Comment:
I think this specificity tooltip will fire off the following (outdated) analytics event:
<img width="298" alt="image" src="https://github.com/chanzuckerberg/single-cell-data-portal/assets/16548075/5d598f87-4a46-4400-8b19-7df8c57c4156">
analytics need to be updated to "effect size" and a new event has to be added for specificity tooltip hover. cc @ainfeld